tests/end_of_line/multiline.js

```diff
@@ -0,0 +1,16 @@
+const foo = bar(
+  (_0: baz) => {
```

Comment:
I can't reproduce the bug in 2.0.2 with this snippet. Please use the original snippet from #6501 as a test instead.

Reply:
Done.

Reply:
The change log entry still contains the old snippet though.